src/edu/ucsd/library/xdre/collection/CollectionHandler.java

```diff
@@ -1172,7 +1172,15 @@ public static String objectPart(String symbol){
 	public static String nextBlankNodeId(){
 		return "_" + (""+random.nextInt()).replace("-", "n");
 	}
-	
+
+	public static boolean isDerivativesRequired(String fid, String use) {
+		if ( StringUtils.isBlank(fid))
+			return false;
+		return (isImage(fid, use) || isDocument(fid, use) || isVideo(fid, use) || isAudio(fid, use))
+				&& ((use == null && fid.startsWith("1.")) 
```

Comment:
@mcritchlow Yes, it sound good and I've broken it up, see https://github.com/ucsdlib/damsmanager/pull/89/commits/1e0d3dc68c38d3b602c40276af13d068224c8b44#diff-352eb836dfc5661ee9af0408149ab1adR1176 .